examples/uganda/uganda/model.py

```diff
@@ -0,0 +1,92 @@
+import math
+import random
+import uuid
+
+import numpy as np
+from mesa import Model
+from mesa.time import RandomActivation
+from shapely.geometry import Point
+
+from mesa_geo.geoagent import GeoAgent
+
+from .space import UgandaArea
+
+
+class Person(GeoAgent):
+    MOBILITY_RANGE_X = 0.0
+    MOBILITY_RANGE_Y = 0.0
+
+    def __init__(self, unique_id, model, geometry, crs, img_coord):
+        super().__init__(unique_id, model, geometry, crs)
+        self.img_coord = img_coord
+
+    def set_random_world_coord(self):
```

Comment:
It seems to be a common pattern to set random coord? Should we package it?

Reply:
Also included in https://github.com/projectmesa/mesa-geo/issues/91.